documentation/homework/week1/src/components/todoList.js

```diff
@@ -0,0 +1,27 @@
+import React,{Component} from 'react';
+import TodoListItem from './todoListItem';
+import TodoAdd from './todoAdd';
+
+class TodoList extends Component {
+
+    render() {
+        const todoList = this.props.listTodo;
+        const todoListItem = todoList.map((data,index) =>
+             <TodoListItem 
```

Comment:
If you run your app and open the dev console in your browser, you'll see this warning:

> Warning: Each child in an array or iterator should have a unique "key" prop.

Did you cover this in week 1 or 2? [Here's](https://reactjs.org/docs/lists-and-keys.html) more info from the React docs.